Source/Python/WarpXWrappers.cpp

```diff
@@ -662,6 +659,10 @@ namespace
         WarpX& warpx = WarpX::GetInstance();
         warpx.FillBoundaryB (warpx.getngE());
     }
+    void warpx_SyncRho () {
+        WarpX& warpx = WarpX::GetInstance();
+        warpx.SyncRho ();
```

Comment:
Since this call site is not a definition or declaration:
```suggestion
        warpx.SyncRho();
```

Reply:
I was matching the format for the functions around this call like `warpx_SyncCurrent`,  `warpx_UpdateAuxilaryData`, etc. Should I change those as well?

Reply:
Oh, dang. In such situations, I am ok to rather match the environment and do a pure stylistic change in a separate PR (if it affects a lot).

Reply:
I'll open a PR later tonight to fix the rest.